### ISSUE TITLE
Taxonomy Queries

### ISSUE THREAD

**Issue by ryanwelcher** [ryanwelcher/advanced-query-loop#35]
I'd like to add some controls for querying by taxonomy. There is prior art from the meta query controls that can be leveraged here.
As MVP, I'll start with being able to use the options laid out here - https://www.whereisit5pmrightnow.com/

I have started some experiments in [this branch](https://github.com/ryanwelcher/advanced-query-loop/tree/feature/taxonomy-control) and am seeing some odd behaviour with the `include_children` and `operator` fields so I may start with basic tax and term selection as MVP.

**Comment by carstingaxion** [comment]
> I'd like to add some controls for querying by taxonomy. There is prior art from the meta query controls that can be leveraged here. As MVP, I'll start with being able to use the options laid out here - https://www.whereisit5pmrightnow.com/
> 
> I have started some experiments in [this branch](https://github.com/ryanwelcher/advanced-query-loop/tree/feature/taxonomy-control) and am seeing some odd behaviour with the `include_children` and `operator` fields so I may start with basic tax and term selection as MVP.

Exactly what I did. I took your meta-query controls and modified them to provided nice controls for taxonomy-queries.

Within my use-case - querying by context, not by hard-coded term-ids, I didn’t had big troubles with the includes property. Maybe I didn’t mention any errors or maybe it could help you going on with this issues. 

Feel invited to https://github.com/carstingaxion/additional-advanced-query-loops/tree/contextual-query-loop and https://github.com/carstingaxion/additional-advanced-query-loops/issues/10

**Comment by JosVelasco** [comment]
Hey, @carstingaxion and @ryanwelcher. I'd love to use the plugin to query related posts by context in a single template.

How can I test the branch for contextual queries? I activated the plugin but could not find the advanced query loop block.

**Comment by carstingaxion** [comment]
> Hey, @carstingaxion and @ryanwelcher. I'd love to use the plugin to query related posts by context in a single template.

Cool @JosVelasco,  that is exactly what I wanted, too and why I forked @ryanwelcher|s great work.

> How can I test the branch for contextual queries? I activated the plugin but could not find the advanced query loop block.

I guess because I was a little too rigid in the beginning. Unsure how far I could get with my idea of contextual queries, I renamed everything to “**Contextual query loop**” to be able to run the original plugin and my fork in parallel. Try this name in the inserter.

Now, that I would like to provide a PR with my additions, I have to revert all the renaming at first. And I haven’t found the time, yet. (And tomorrow morning starts the family holiday)

So feel free to use my fork until some point in the future, where contextual queries will eventually be merged into _Advanced query loop._

**Comment by JosVelasco** [comment]
Unfortunately, the block does not appear. I wonder if it has something to do with a newer WordPress version.

**Comment by carstingaxion** [comment]
> Unfortunately, the block does not appear. I wonder if it has something to do with a newer WordPress version.

Fixed it.

The original plugin _gitignores_ the `/build` files, why things were working for me all the time, but for nobody else (who did not do the _build_ manually). **I'm sorry.**

Now it should work for you, as it [works in a Playground](https://playground.wordpress.net/?mode=seamless#{%22$schema%22:%22https://playground.wordpress.net/blueprint-schema.json%22,%22landingPage%22:%22https://playground.wordpress.net/wp-admin/post-new.php%22,%22preferredVersions%22:{%22php%22:%228.2%22,%22wp%22:%22latest%22},%22phpExtensionBundles%22:[%22kitchen-sink%22],%22features%22:{%22networking%22:true},%22steps%22:[{%22step%22:%22login%22,%22username%22:%22admin%22,%22password%22:%22password%22},{%22step%22:%22installPlugin%22,%22pluginZipFile%22:{%22resource%22:%22url%22,%22url%22:%22https://github-proxy.com/proxy/?repo=carstingaxion/additional-advanced-query-loops&branch=contextual-query-loop%22}}]}) 

_(funfact: In 6 hours my family & me are going to start our holiday, so we can both be lucky to have this fixed before)_ 

Thanks for your feedback @JosVelasco !

**Comment by JosVelasco** [comment]
Happy holidays, @carstingaxion!

**Comment by ryanwelcher** [comment]
Closed with #90
